src/cli/subcommand/plan.rs

```diff
@@ -13,7 +13,12 @@ use crate::cli::CommandExecute;
 pub struct Plan {
     #[clap(subcommand)]
     pub planner: Option<BuiltinPlanner>,
-    #[clap(env = "NIX_INSTALLER_PLAN", default_value = "/dev/stdout")]
+    /// Where to write the generated plan (in JSON format)
+    #[clap(
+        long = "plan-file",
```

Comment:
Open to bikeshedding on the flag name. `--out` seemed too terse, and `--out-file` seemed not quite right given the env var.

Also open to changing the env var in line with whatever the flag ends up being, since I think `NIX_INSTALLER_PLAN` could be misread as "the plan to use" (which doesn't really make sense but in this context).

Reply:
We should change the environment var to match.

I think `--out`, `--output`, `--out-file` are all fine. This isn't a command we really want regular users running without having an idea what it's for anyways... It's mostly for testing/validation.

Reply:
OK, done. I chose `--out-file`.